common/changes/@rushstack/module-minifier-plugin/webpack5-minifier-plugin_2022-05-18-20-54.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/module-minifier-plugin",
+      "comment": "Factor out minifiers into @rushstack/module-minifier.",
+      "type": "patch"
```

```suggestion
      "type": "minor"
```

In case anything goes wrong.